src/discof/replay/fd_replay_tile.c

```diff
@@ -1223,58 +1210,21 @@ handle_new_slot( fd_replay_tile_ctx_t * ctx,
                  ulong                  slot,
                  ulong                  parent_slot ) {
 
-  /* We need to handle logic that creates a bank and funk txn since
-     we are starting to execute a new slot. We must also manage the
-     forks data structure to reflect that this slot is now being
-     executed. */
-
-  /* First, update fd_forks_t */
-
-  /* Make sure that the slot is not already in the frontier. */
-  if( FD_UNLIKELY( fd_fork_frontier_ele_query(
-      ctx->forks->frontier,
-      &slot,
-      NULL,
-      ctx->forks->pool ) ) ) {
-    FD_LOG_CRIT(( "invariant violation: child slot %lu was already in the frontier", slot ) );
-  }
+  /* We need to execute a new block. */
 
-  /* This means we want to execute a slice on a new slot. This means
-      we have to update our forks and create a new bank/funk_txn. */
-  fd_fork_t * fork = fd_forks_prepare( ctx->forks, parent_slot );
-  if( FD_UNLIKELY( !fork ) ) {
-    FD_LOG_CRIT(( "invariant violation: failed to prepare fork for slot: %lu", slot ));
+  fd_bank_t * bank = fd_banks_get_bank( ctx->banks, slot );
+  if( FD_UNLIKELY( !!bank ) ) {
+    FD_LOG_CRIT(( "invariant violation: block %lu (parent_block %lu) already has a bank", slot, parent_slot ) );
   }
```

Comment:
can you add a documentation note this currently does not properly handle the minority fork publish issue until ref cnt publishing is implemented?